src/jit/compiler.h

```diff
@@ -1140,6 +1140,7 @@ struct fgArgTabEntry
     SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASSING_DESCRIPTOR structDesc;
 #endif // defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
 
+#ifdef _TARGET_ARM_
```

Comment:
I didn't want to have three full sets of #ifdefs, for three different methods, which would have been a lot of clutter, so I did it this way instead.